gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,19 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-28245   
**Check Name:** KaTeX is a JavaScript library for TeX math rendering on the web. KaTeX ...   
**Severity:** MEDIUM   
**Fixed Version:** 0.16.10   
**Reachable Path(s) Found:** No  
**Description:** KaTeX is a JavaScript library for TeX math rendering on the web. KaTeX users who render untrusted mathematical expressions could encounter malicious input using `\includegraphics` that runs arbitrary JavaScript, or generate invalid HTML. Upgrade to KaTeX v0.16.10 to remove this vulnerability.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-28245